### PR TITLE
send jupyter errors through ansi processing

### LINKED ISSUE #4204: When an error is rendered in the output, the ascii escape codes should be stripped or processed to HTML
### Bug description

Currently when an `error` is rendered in the output, the ascii escape codes are also printed out

image

This is what it looks like in the terminal:

image

It would be nice if either quarto

1) starts julia with the flag `--color=no`. This is the same output in the terminal with that flag

image

OR

2) parses ascii codes in outputs using a javascript library to generate color html output. For example, you could vendor this: https://github.com/drudru/ansi_up

Reproducible example:

Create a `_quarto.yml` file:

```
project:
  type: website
  title: "MethodError"
  output-dir: _output
  execute-dir: project

execute:
  output: true
  echo: true
  warning: true
  error: true
  freeze: auto
  eval: true
  cache: true

jupyter: julia-1.8
```

Create a `index.qmd` file:

````md
---
title: Method error
---

#### MethodError

```{julia}
ceil(1.2 + 2.3im)
```
````

Version information:

```julia
julia> versioninfo()
Julia Version 1.8.5
Commit 17cfb8e65e (2023-01-08 06:45 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 8 × 11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, tigerlake)
  Threads: 1 on 8 virtual cores
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

### PR DESCRIPTION
**Pull request by cscheid**
Closes #4204.

### CODE REVIEW

File: src/core/jupyter/jupyter.ts [hunk 1]
```diff
@@ -1719,8 +1719,22 @@ function mdOutputStream(output: JupyterOutputStream) {
...
-  return mdCodeOutput([output.ename + ": " + output.evalue]);
+async function mdOutputError(
+  output: JupyterOutputError,
+  options: JupyterToMarkdownOptions,
+) {
+  if (!options.toHtml || !hasAnsiEscapeCodes(output.evalue)) {
+    return mdCodeOutput([output.ename + ": " + output.evalue]);
+  }
+  const html = await convertToHtmlSpans(output.evalue);
+  console.log(output.evalue);
```
Comment (kdheepak) [inline src/core/jupyter/jupyter.ts:1730]: Is this `console.log(output.evalue)` required?

Comment (cscheid) [reply src/core/jupyter/jupyter.ts:1730]: Nope, that's me being stupid. Thanks for the catch.

Comment (cscheid) [reply src/core/jupyter/jupyter.ts:1730]: Fixed on `main`.

Comment (kdheepak) [reply src/core/jupyter/jupyter.ts:1730]: Awesome thanks!

**Pull request merged**